proposals/Level-1.md

```diff
@@ -78,94 +77,69 @@ instruction. Thrown exceptions are handled as follows:
-index space](#event-index-space). Each event index refers to a corresponding
-runtime tag that uniquely identifies the corresponding event to be handled.  The
-corresponding runtime tag is called an `event tag`.
-
-Event types are numbered numerically, starting with the import section, and then
-followed by event types in the event section.  Each event type within a module
-are associated with unique event tags.
+The type signature of an event type is an index to the corresponding function
+signature defined in the `type` section. The parameter of the function signature
+defines the list of values associated with the event. The return value(s) of the
+signature defines what is passed back to yeilding instruction so that execution
+may resume.
 
-Event types can also be exported by adding appropriate descriptors in the export
-section of the module. Each imported/exported event index must be named to
-reconcile the corresponding (runtime) event tag.
+Within a module, event types are identified by an `event index` to the [event
+index space](#event-index-space). This (module specific) index identifies the
+corresponding identifie the event type to apply.
 
 ### Exceptions
 
-An `exception` is a WebAssembly event that can be thrown and caught.  That is,
-an `exception type` is an event type whose event specifier is 0.  The `type
-signature` of the exception is the corresponding type signature of the event
-type. That is, the parameter list of the type signature defines the list of
-values associated with the exception, and the result type must be type `void`.
-
-Exceptions are identified by the by the corresponding event index, and is called
-an `exception index`. The corresponding (runtime) event tag is called the
-`exception tag`.  Like all event types, each exception type within a module
-(i.e. in the event and import sections) are associated with unique exception
-tags.
+An `exception` is an internal construct in WebAssembly that is maintained by the
+embedder. WebAssembly exceptions (as opposed to embedder exceptions) are defined
+in the `event` and import sections of a module. Each event type with the
+event specifier 0 defines an `exception type`. The event index is also
+called the `exception index`. Similarly, the corresponding event tag is 
+called an `exception tag`.
 
 Exception indices are used by:
 
-1. The `throw` instruction which creates a WebAssembly exception with the
-   corresponding exception tag, and then throws it.
+1. The `throw` instruction which creates a WebAssembly exception
+   with the corresponding exception tag, and then throws it.
 
-2. The `if_except` instruction that queries an exception to see if the exception
-   tag corresponding to the module's exception index. If true it pushes the
-   corresponding values of the exception onto the stack.
+2. The `if_except` instruction queries a exception to see if the corresponding
+   exception tag of the exception matches the wanted exception index.  If true
```

Comment:
"matches the _tag of_ the wanted exception index."

Just to be explicit that we are comparing tags to tags, and not tags to indices.

Reply:
"matches the tag denoted by the exception index"